package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.3.13",
+    "@ceramicnetwork/cli": "^0.22.0",
```

Comment:
Does the cli package pull in the http-client? I believe the http-client is all that is needed.  `@ceramicnetwork/http-client`

Reply:
Using only that particular package (http), to avoid confusion